src/MutantRunResultMatcher.ts

```diff
@@ -26,12 +26,26 @@ export default class MutantRunResultMatcher {
     });
   }
 
-  private mutantCovers(mutant: Mutant, location: Location): boolean {
+  private mutantCoversFile(mutant: Mutant, coveredFile: CoverageResult): boolean {
+    let smallestStatementFound = "";
+    
+    Object.keys(coveredFile.statementMap).forEach(statementId => {
+      let location = coveredFile.statementMap[parseInt(statementId)];
+        
+      if(this.mutantCoversLocation(mutant, location)){
+        smallestStatementFound = statementId;
```

Comment:
I'll fix this in case we ever come across a coverage reporter which doesn't report coverage as a tree